### PR TITLE
Design and layout cleanup

### PR DESCRIPTION
**Pull request by ddabble**
### Highlights:
* Fixed weird scaling and small font sizes on mobile (c2322d7be2ddab5ea1c054b19153d297ab7f8801) and fixed some related sizing and layout issues (a1998f885858e2d4b485bc27fbc34b354b23eca0)
* Made the Ubuntu font apply everywhere (54f6ac42debc3ea69f48fa9370a46a14081a5b5a)
* Made the usage of the colors of MAKE's graphic charter consistent (5b7dcc378943c905c818031e8b3355a0b5ae8fc5)
* Improved design of committee and equipment admin (e98ff98d7cedbeaade2253fccd8fb5925552cb8f)
* Made the usage of edit/save/cancel buttons consistent (1fc65d9a17d550510f5f155eda1cb1ae3d9ed13b)

### Also, code improvements and bug fixes:
* Huge cleanup of various whitespace, which includes cleaning up *actual* whitespace, but is mostly rearrangement of code (that doesn't alter the function in any way) (412e53f9af6e51bc4b9b075bf2986b59283d6136)
* Added missing priority field to machine edit form (329355c3c83614df85814eb8b78e0fd0c055b01a)
* Fixed `CategoryAdmin` missing from Django admin (52e72fe93059d5f588cab8c24348e077d9220387)

### CODE REVIEW

**Review by Maisergodt — CHANGES_REQUESTED** [review]
Nice!
I left some feedback :)

File: make_queue/models/models.py [hunk 1]
```diff
@@ -241,17 +247,43 @@ def can_create_reservation(reservation):
...
     start_time = models.DateTimeField()
     end_time = models.DateTimeField()
-    event = models.ForeignKey(TimePlace, null=True, blank=True, on_delete=models.CASCADE)
+    event = models.ForeignKey(
+        to=TimePlace,
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+    )
     showed = models.NullBooleanField(default=None)
```
[the remark this answers is not in the document]
Comment (ddabble) [reply make_queue/models/models.py:268]: Yes, it should 😄 I have a fix for it ready in an upcoming PR 🙂

File: faq/models.py [hunk 2]
```diff
@@ -5,25 +5,15 @@
 
 
 class Category(models.Model):
-    name = MultiLingualTextField(
-        unique=True,
-        verbose_name=_('Category'),
-    )
+    name = MultiLingualTextField(unique=True, verbose_name=_('Category'))
```
[the remark this answers is not in the document]
Comment (ddabble) [reply faq/models.py:8]: I agree, though I'm unsure whether this change (and similar changes elsewhere in the codebase) fits this PR.. 🤔 Also I have the same fix ready for an(other) upcoming PR 😁

File: internal/templates/internal/header.html [hunk 3]
```diff
@@ -25,19 +25,19 @@
     <nav id="nav" class="ui secondary huge inverted menu" aria-label="Main">
         <a class="item" href="{% url "members" %}">
             <div class="text">{% trans "Members" %}</div>
-            <div class="make_bg_blue bubble-background"></div>
+            <div class="make-bg-blue bubble-background"></div>
         </a>
          <a class="item" href="{% url "secrets" %}">
             <div class="text">{% trans "Secrets" %}</div>
-            <div class="make_bg_blue bubble-background"></div>
+            <div class="make-bg-blue bubble-background"></div>
```
[the remark this answers is not in the document]
Comment (ddabble) [reply internal/templates/internal/header.html:32]: I personally think the current color pattern is better than any randomized pattern 😅 There exists a [card on Trello](https://trello.com/c/WMEziJ6E/182-add-an-admin-panel-for-editing-the-header-buttons) that details adding an admin panel for changing the colors of these buttons - among other things. Changing any of this, however, is kind of not within the scope of this PR, in my opinion.. Do you agree? 🙂

[the remark this answers is not in the document]
Comment (Maisergodt) [reply internal/templates/internal/header.html:32]: I wasn't aware of that card, then nevermind my comment 😛

File: news/templates/news/admin_event_ticket_list.html [hunk 4]
```diff
@@ -15,20 +16,21 @@ <h1>{{ event.title }}</h1>
             <i class="ui angle double left icon"></i>{% trans "Event admin page" %}
         </a>
 
-        <h3>{% trans "Emails" %}
+        <h3>
+            {% trans "Emails" %}
             <button class="copy-token"><i class="clipboard icon"></i></button>
```
[the remark this answers is not in the document]
Comment (ddabble) [reply news/templates/news/admin_event_ticket_list.html]: Yes, good idea! 💯

**Comment by ddabble** [comment]
@Maisergodt I improved the design of the button for copying the email addresses of event ticket holders, as you suggested 😊

**Review by sigtheidiot — APPROVED** [review]

**Review by Maisergodt — APPROVED** [review]

**Pull request merged**